WeakAuras/GenericTrigger.lua

```diff
@@ -149,6 +149,14 @@ end
 function TestForMultiSelect(trigger, arg)
   local name = arg.name;
   local test;
+  if trigger[name] and type(trigger[name]) ~= "table" then -- lazy migration for type converted from select to multiselect
```

Comment:
Is this a saved data migration? I'd *really* prefer that these are all done during ADDON_LOADED or via WeakAuras.Modernize. what is your reasoning for doing it here?

Reply:
i was lazy i'll move it